src/test/kotlin/com/igorwojda/list/formattrainroute/challenge.kt

```diff
@@ -4,7 +4,10 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun formatTrainRoute(stations: List<String>): String {
-    TODO("not implemented")
+    return "Train is calling at " + if (stations.size > 1) stations.subList(0, stations.size - 1)
```

Comment:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/join-to-string.html is a very powerful tool